src/code.sh

```
@@ -40,14 +40,14 @@ _set_env() {
         blacklist_file \
         known_fusions \
         sample_name \
-        star_fusion_bam
+        star_bam
 
     # Extract CTAT library filename
     lib_dir=$(find /home/dnanexus/genome_lib -type d -name "*" -mindepth 1 -maxdepth 1 | rev | cut -d'/' -f-1 | rev)
 
     # Extract sample name from input_bam
     sample_name=$(echo $bam_prefix | cut -d '.' -f 1)
```

Shellcheck (0.10.0)

[warning] bam_prefix is referenced but not assigned.

(SC2154)